### PR TITLE
[kai/permitted-individual-create] Other Permitted Individual service

### PR DESCRIPTION
**Pull request by kai-wei-mo**
## Notion ticket link

[Create new permitted individual](https://www.notion.so/uwblueprintexecs/Create-new-permitted-individual-3c01f8c61da24aa7adb730ab99c54635)

## Implementation description
* Implemented as the ticket describes

## Steps to test
- unit tests

## What should reviewers focus on?
- N/A

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/python/app/rest/other_permitted_individual_routes.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from flask import Blueprint, current_app, jsonify, request
```
Comment (kai-wei-mo) [inline backend/python/app/rest/other_permitted_individual_routes.py:1]: this file will be needed in a future ticket i have

File: backend/python/app/middlewares/validate.py [hunk 2]
```diff
@@ -13,6 +17,7 @@
     "RegisterUserDTO": RegisterUserDTO,
     "UpdateUserDTO": UpdateUserDTO,
     "CreateCaregiverDTO": CreateCaregiverDTO,
+    "CreateOtherPermittedIndividualDTO": CreateOtherPermittedIndividualDTO,
```
Comment (jennchenn) [inline backend/python/app/middlewares/validate.py:20]: is this still being used?

**Review by jennchenn — COMMENTED** [review]
do you mind just updating the PR description and title to reflect the removal of route changes?

File: backend/python/app/middlewares/validate.py (hunk 2 above)
Comment (kai-wei-mo) [reply backend/python/app/middlewares/validate.py]: my bad, removed it now

**Pull request merged**